examples/replicate.go

```diff
@@ -5,8 +5,8 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/blind-oracle/pgoutput"
```

Comment:
I know this is useful for testing, but this needs to be reverted before merging.